src/less/components/progress-tracker.less

```diff
@@ -0,0 +1,142 @@
+            a,
+            span {
+                color: @brand-primary;
+                padding: @breadcrumb-padding;
+                background-color: @breadcrumb-bg;
+                border: 1px solid @breadcrumb-border-color;
+                border-right: none;
+                position: relative;
+                text-transform: uppercase;
+                margin-bottom: 0.2rem;
+
+                &:before,
+                &:after {
+                    left: 100%;
+                    top: 50%;
+                    border: solid transparent;
+                    content: "";
+                    height: 0;
+                    width: 0;
+                    position: absolute;
+                    pointer-events: none;
+                }
+
+                &:after {
+                    border-left-color: @breadcrumb-border-color;
+                    border-width: @breadcrumb-arrow-border;
+                    margin-top: -@breadcrumb-arrow-border;
+                    z-index: 1;
+                }
+
+                &:before {
+                    border-left-color: @breadcrumb-bg;
+                    border-width: @breadcrumb-arrow-bg;
+                    margin-top: -@breadcrumb-arrow-bg;
+                    z-index: 2;
+                }
+            }
+
+            &:first-child {
+                a,
+                span {
+                    padding-left: @breadcrumb-padding-first;
+                }
+            }
+
+            &.active {
+                a,
+                span {
+                    color: @breadcrumb-active-color;
+                    background-color: @breadcrumb-active-bg;
+
+                    &:before {
+                        border: none;
+                    }
+                }
+            }
+
+            &:not(.active) {
+                span {
+                    color: @breadcrumb-disabled-color;
+                    pointer-events: none;
+                    background-color: @breadcrumb-disabled-bg;
+
+                    &:before {
+                        border-left-color: @breadcrumb-disabled-bg;
+                    }
+                }
+            }
+        }
+
+        &.progress-tracker-sm {
+            li {
+                a,
+                span {
+                    padding: @breadcrumb-padding-sm;
+                    font-size: 80%;
+
+                    &:after {
+                        border-width: @breadcrumb-arrow-border-sm;
+                        margin-top: -@breadcrumb-arrow-border-sm;
+                    }
+
+                    &:before {
+                        border-width: @breadcrumb-arrow-bg-sm;
+                        margin-top: -@breadcrumb-arrow-bg-sm;
+                    }
+                }
+
+                &:first-child {
+                    a,
+                    span {
+                        padding-left: @breadcrumb-padding-first-sm;
+                    }
+                }
+            }
+        }
+    }
+}
```

Comment:
Nice, this fixes a lot of tiny glitches! Slightly suboptimal on IE though :(. One question: Could this file perhaps inherit or use a shared css "generator" function with the breadcrumbs.css?